client/router_create.go

```diff
@@ -1180,6 +1180,14 @@ sasldb_path: /tmp/qdrouterd.sasldb
 			return err
 		}
 	}
+
+	// Create a CA associated with the site to create certificates for each service
+	// and a generic client secret with that CA.
+	err = cli.CreateServiceCA(siteOwnerRef)
```

Comment:
An alternative to consider would be to add the ServiceCaSecret to the van.CertAuthoritys so that the CA gets created in a fashion similar to the others in the van definition.

Reply:
Thanks, this is very helpful. The CA for the skupper site is now created with the rest of the CAs that Skupper manages.